src/requests/requester.rs

```diff
@@ -22,43 +23,43 @@ pub trait Requester {
 
     type GetUpdates: Request<Payload = GetUpdates, Err = Self::Err>;
 
-    /// For telegram documentation see [`GetUpdates`]
+    /// For Telegram documentation see [`GetUpdates`].
```

Comment:
Open an issue, I'll fix it later.

https://github.com/teloxide/teloxide-core/blob/e260f11558e5bfae89b4bfa58cd3d99a487da1ab/src/requests/requester.rs#L19-L21